pkg/ottl/contexts/ottldatapoint/README.md

```diff
@@ -49,6 +49,13 @@ The following paths are supported.
 | datapoint.scale                                | the scale of the data point being processed                                                                                                                                         | int64                                                                   |
 | datapoint.zero_count                           | the zero_count of the data point being processed                                                                                                                                    | int64                                                                   |
 | datapoint.quantile_values                      | the quantile_values of the data point being processed                                                                                                                               | pmetric.SummaryDataPointValueAtQuantileSlice                            |
+| context.client.addr                            | the remote address string from the client info                                                                                                                                      | string                                                                  |
```

Comment:
We can add these as link to the `context` docs, so we don't need to keep it in sync? similar to the `span.*` on https://github.com/ronkorland/opentelemetry-collector-contrib/blob/main/pkg/ottl/contexts/ottlspanevent/README.md